pkg/crypto/ccm/ccm_test.go

```diff
@@ -18,8 +19,10 @@ func mustHexDecode(s string) []byte {
 	return r
 }
 
-var aesKey1to12 = mustHexDecode("c0c1c2c3c4c5c6c7c8c9cacbcccdcecf")
-var aesKey13to24 = mustHexDecode("d7828d13b2b0bdc325a76236df93cc6b")
+var (
+	aesKey1to12  = mustHexDecode("c0c1c2c3c4c5c6c7c8c9cacbcccdcecf") //nolint:gochecknoglobals
+	aesKey13to24 = mustHexDecode("d7828d13b2b0bdc325a76236df93cc6b") //nolint:gochecknoglobals
+)
```

Comment:
If I convert this to a func would it recalculate every call? 
Keep as global to reuse variable?

Reply:
I think that is the right choice! I don't think it is so bad for tests.

For main package code unexpected things can happen (like `append` will modify) so want to avoid those sharp edges.